Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/commands/CommandDumpSigns.java

```diff
@@ -75,7 +75,10 @@ private void dumpAtPos(AIHelper helper, PrintStream out, BlockPos p) {
 					out.print(p.getY());
 					out.print("\t");
 					out.print(p.getZ());
-					ITextComponent[] texts = ((SignTileEntity) tileentity).signText;
+					ITextComponent l1 = ((SignTileEntity) tileentity).getText(1);
```

Comment:
Hasn't a sign 4 lines?

Reply:
It does, I did 1 2 3 because I didn't know if it started from 0 :D